### ISSUE TITLE
MissingMethod error in AddFunds

### ISSUE THREAD

**Issue by IceRaptor** [IceRaptor/IRTweaks#19]
Reported in RT #ticket-3285:

Hello, my issue is that when I attempt to buy an item from a shop that has multiples of that item, after I select the amount I want and click buy nothing happens.
Buying single items from the shop works fine though.
Also this happened right after I updated to version 1.2.

```
BattleTech.UI.SG_Stores_MultiPurchasePopup.ReceiveButtonPress:
        Prefixes:
                us.frostraptor.IRTweaks
@FrostRaptor
MissingMethodException: void BattleTech.SimGameState.AddFunds(int,string,bool)
  at BattleTech.UI.SG_Stores_MultiPurchasePopup.OnConfirm () [0x00008] in <4184af8dbeb44635831353f4d349631c>:0
  at (wrapper dynamic-method) BattleTech.UI.SG_Stores_MultiPurchasePopup.ReceiveButtonPress_Patch1(object,string)
```

**Comment by IceRaptor** [comment]
[ModTek.log](https://github.com/IceRaptor/IRTweaks/files/4370220/ModTek.log)
[cleaned_output_log (1).txt](https://github.com/IceRaptor/IRTweaks/files/4370221/cleaned_output_log.1.txt)
[output_log (1).txt](https://github.com/IceRaptor/IRTweaks/files/4370222/output_log.1.txt)

**Comment by IceRaptor** [comment]
HBS added new optional params to AddFunds in 1.9.1 - just needed a recompile.

Fixed in 0.6.3
